packages/compass-query-bar/src/components/query-option/query-option.jsx

```diff
@@ -125,7 +125,7 @@ class QueryOption extends Component {
         className={_className}
         data-test-id="query-bar-option">
         <div
-          className={classnames(styles.label)}
```

Comment:
For what this classnames was used here before?

Reply:
It wasn't doing anything, there are a lot of occurrences in Compass of this, I try to remove them when making changes in the area. The places where `classnames` has a use are when classes need to be concatenated or resolved, places where it's wrapping a single class don't do anything (maybe a super slight perf upgrade by removing them).